apps/homescreen/test/marionette/bookmark_edit_test.js

```diff
@@ -38,7 +38,8 @@ marionette('Homescreen - Bookmark Edit', function() {
     client.switchToFrame(system.getHomescreenIframe());
   });
 
-  test('pressing enter after editing the bookmark', function() {
+  // Skipping until UX input on editing Pinned sites
+  test.skip('pressing enter after editing the bookmark', function() {
```

Comment:
We need to do something other than just skip these tests. You should still be able to edit a pinned site (bookmark) using the bookmarks app, I don't think that feature is going away.